### ISSUE TITLE
Harvester?

### ISSUE THREAD

**Issue by chinabala** [madMAx43v3r/mmx-node#24]
Do we have harvester for mmx at all?
Something like /CA for Chia. Or a better way to implement.

**Comment by madMAx43v3r** [comment]
You mean remote harvester?

**Comment by chinabala** [comment]
Yes. So that we don't have to run full node and wallet on every machine.
Also, when node/wallet db gets larger and larger, it will be a nightmare for new miners.

**Comment by madMAx43v3r** [comment]
See readme, `./run_harvester.sh ...`

**Comment by chinabala** [comment]
Thank you! you're amazing:)
